ToNote/UI/ExpanderStyles.xaml

```diff
@@ -0,0 +1,107 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:local="clr-namespace:ToNote.UI">
+
+    <Style x:Key="ExpanderDownHeaderStyle" TargetType="{x:Type ToggleButton}">
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="{x:Type ToggleButton}">
+                    <Border Padding="{TemplateBinding Padding}">
+                        <Grid Background="Transparent" SnapsToDevicePixels="False">
+                            <Grid.ColumnDefinitions>
+                                <ColumnDefinition Width="20"/>
+                                <ColumnDefinition Width="*"/>
+                            </Grid.ColumnDefinitions>
+                            <Path x:Name="arrow" RenderTransformOrigin="0.5,0.5" Data="M 1,1.5 L 4.5,5 8,1.5" HorizontalAlignment="Center" SnapsToDevicePixels="false" Stroke="{StaticResource DefaultToggleButtonForegroundBrush}" StrokeThickness="2" VerticalAlignment="Center" >
+                                <Path.RenderTransform>
+                                    <RotateTransform Angle="0"/>
+                                </Path.RenderTransform>
+                            </Path>
+                            <ContentPresenter Grid.Column="1" HorizontalAlignment="Left" Margin="0,0,0,0" RecognizesAccessKey="True" SnapsToDevicePixels="True" VerticalAlignment="Center"/>
+                            <!--Make binding text for versatility-->
+                            <TextBlock x:Name="text" Text="Tags" Grid.Column="2" Foreground="{StaticResource DefaultToggleButtonForegroundBrush}" FontSize="12"/>
```

Comment:
Remove this line. The header content should be passed to the Expander through Header attribute. The Content Presenter thats is on line 20 displays it automagically.